gems/pending/openstack/openstack_handle/pagination/base.rb

```diff
@@ -0,0 +1,24 @@
+module OpenstackHandle
+  module Pagination
+    class Base
+      def initialize(service, os_handle, collection_type, options = {}, method = :all)
+        @service         = service
+        @os_handle       = os_handle
+        @collection_type = collection_type
+        @options         = options
+        @method          = method
+      end
+
+      private
+
+      def call_list_method(collection_type, options, method,
+                           paginate_options = {:limit => @service.default_pagination_limit})
+        @service.public_send(collection_type).public_send(method, options.merge(paginate_options))
+      end
+
+      def marker(last_server, collection_type)
```

Comment:
Should this `marker` method be in the `Marker` class?  For instance, why would the `Pagination::None` class need a `marker` method if it doesn't have any pagination?